react/FieldMessage/FieldMessage.js

```diff
@@ -1,31 +1,34 @@
+// @flow
 import styles from './FieldMessage.less';
 
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
 
 import classnames from 'classnames';
 
 import ErrorIcon from '../ErrorIcon/ErrorIcon';
 import TickCircleIcon from '../TickCircleIcon/TickCircleIcon';
 
 import Text from '../Text/Text';
-
-export default class FieldMessage extends Component {
+import { TONE } from '../private/tone';
+
+type Props = {
+  id: string,
+  invalid?: boolean,
+  help?: string,
+  helpProps?: Object,
+  valid?: boolean,
+  message: false | Node | string,
```

Comment:
keep forgetting it, but Node should already include 2 other cases 🤔

Reply:
It seems that it's not quite the same, I changed it to `message: Node` and got this error.

![screen shot 2018-09-25 at 5 02 33 pm](https://user-images.githubusercontent.com/1896277/45998041-dd23fd80-c0e4-11e8-82bd-bb31213c5731.png)